docker-compose.override.yml

```diff
@@ -1,9 +1,9 @@
 
 version: '2'
 
-volumes:
-  emberosf-sync:
-    external: true
+#volumes:
+#  emberosf-sync:
+#    external: true
```

Comment:
@brianjgeiger FYI: Anyone who pulled today will fail on `docker-compose up` until this change goes in.